container_files/mldb_nginx_site.conf

```diff
@@ -87,12 +87,20 @@ server {
             return 301 $scheme://$http_host{{HTTP_BASE_URL}}/doc/;
         }
 
-        location {{HTTP_BASE_URL}}/doc/ {
-            # send documentation render requests to mldb (.md.html)
-            rewrite ^{{HTTP_BASE_URL}}(/?.*) $1 break;
-            try_files $uri $uri/ @mldb;
+        location = {{HTTP_BASE_URL}}/resources {
```

Comment:
Not sure if this is required. That rule was in place so that people hitting `/doc` would get redirected correctly to `/doc/` with the correct port number in the URL.
But I don't think there is (or should be) any link pointing to `/resources` directly.